server/pulp/server/webservices/views/task_groups.py

```diff
@@ -32,6 +33,24 @@ def get(self, request, group_id):
         """
         raise MissingResource(group_id)
 
+    @auth_required(authorization.DELETE)
+    def delete(self, request, group_id):
+        """
+        Dispatch tasks.cancel to delete tasks in a single task_group.
+
+        :param request: WSGI request object
+        :type  request: django.core.handlers.wsgi.WSGIRequest
+        :param group_id: The ID of the task group you wish to cancel
+        :type  group_id: basestring
+
+        :return: Response containing None
+        :rtype:  django.http.HttpResponse
+        """
+        raw_tasks = TaskStatus.objects(group_id=group_id)
```

Comment:
Since you only need one field, use the "only" method: http://docs.mongoengine.org/apireference.html#mongoengine.queryset.QuerySet.only